scripts/set-app-template-versions.js

```diff
@@ -36,28 +42,42 @@ const setVersion = (template, rootTmpDir) => {
   const cloneUrl = `${CLONE_URL_PREFIX}-${template}`;
   var cmd = `git clone ${cloneUrl}`;
 
-  console.log(`Setting zapier-platform-core version to ${newVersion} in ${template} app template.`);
+  console.log(`Setting versions of node, npm, and zapier-platform-core to ${newVersions.nodeVersion}, ${newVersions.npmVersion}, and ${newVersions.coreVersion} respectively in ${template} app template.`);
   console.log(`cloning ${cloneUrl}\n`);
 
   return exec(cmd, rootTmpDir)
     .then(() => {
       const packageJsonFile = path.resolve(rootTmpDir, `${repoName}/package.json`);
       const packageJson = require(packageJsonFile);
 
-      if (packageJson.dependencies['zapier-platform-core'] === newVersion) {
+      const nvmrcFile = path.resolve(rootTmpDir, `${repoName}/.nvmrc`);
+      const nvmrcNodeVersion = fse.readFileSync(nvmrcFile, 'utf8').trim().substr(1); // strip off leading 'v'
+
+      const travisYamlFile = path.resolve(rootTmpDir, `${repoName}/.travis.yml`);
+      const travisYaml = yaml.load(travisYamlFile);
+
+      const nodeVersion = semver.Comparator(newVersions.nodeVersion).semver.version
+      if (packageJson.dependencies['zapier-platform-core'] === newVersions.coreVersion && packageJson.engines['node'] === newVersions.nodeVersion && packageJson.engines['npm'] === newVersions.npmVersion && nvmrcNodeVersion === nodeVersion && travisYaml.node_js[0] === nodeVersion) {
```

Comment:
should probably move this into a function (like `hasCurrentVersions`). Reusable, plus it'll be easy to put it on multiple lines to be more clear about what's being checked. 

```javascript
return (
  packageJson.dependencies['zapier-platform-core'] === newVersions.coreVersion &&
  packageJson.engines['node'] === newVersions.nodeVersion && 
  // ... 
)
```